README.md

```diff
@@ -136,4 +140,21 @@ Repeat this procedure for every custom action in the file:
 6. Copy the whole content of the `lke-configurations/plugins.json` file **except** for the `{` at the beginning and for the `}` at the end of the file
 7. On the `Plugin settings` field, add an new empty line before the last `}`, add in this line a comma (`,`) and, after the comma, paste the content copied in the step 6
 8. Click on `Save`
-9. Done!
\ No newline at end of file
+9. On Linkurious Enterprise dashboard, go to `Admin` -> `Data-source Schema`
+10. Scroll to the end of `NODES CATEGORIES & EDGES TYPES` list and click on `Create new categories`
+![](/assets/img/IMG_08.png)
```

Comment:
Indent this under `Scroll` (i.e. 4 spaces) since the image has to be within the list. Even though this doesn't affect the image, it's more readable and avoids issues if we add more text later on